ui/cmd/nexui/main.go

```diff
@@ -1,10 +1,70 @@
 package main
 
-import nexui "github.com/synadia-io/nex/ui"
+import (
+	"log/slog"
+	"os"
+	"strconv"
+
+	nexui "github.com/synadia-io/nex/ui"
+)
+
+const (
```

Comment:
Sorry for the nit- I think we may want to audit all of the `const` throughout the codebase to use camelcase.

Reply:
i can live with that